src/app/projects/layout.tsx

```diff
@@ -0,0 +1,27 @@
+import '../globals.css'
+import "@fortawesome/fontawesome-svg-core/styles.css";
+import { config } from "@fortawesome/fontawesome-svg-core";
+import type { Metadata } from 'next'
+import Provider from './provider';
+config.autoAddCss = false;
+
+export const metadata: Metadata = {
+  title: 'Froggy Friends',
+  description: 'Submit Project',
+}
+
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode
+}) {
+  return (
+    <html lang="en">
+      <body className='font-hwt-artz'>
+        <Provider>
+          {children}
+        </Provider>
+      </body>
+    </html>
```

Comment:
same thing here, you should be able to change this to 

```suggestion
        <Provider>
          {children}
        </Provider>
```